doc/nrf/protocols/sidewalk/index.rst

```diff
@@ -0,0 +1,24 @@
+.. _ug_sidewalk:
+
+Sidewalk
+########
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. caution::
+   Sidewalk protocol is owned and licensed by Amazon.
+   Before continuing with the nRF Connect SDK for Sidewalk, you must familiarize yourself with the official `Amazon license`_, including all the applicable restrictions.
+   Please note, that redistributing Sidewalk under the nRF Connect SDK is prohibited.
+
+   Be aware, that by downloading and including the Sidewalk repository, you accept the above-mentioned license.
+
+   Due to these restrictions, Sidewalk cannot be automatically fetched through the nRF Connect SDK and will require performing a dedicated setup.
```

Comment:
I would add a link pointing to the documentation describing beforementioned dedicated setup e.g.: "You can access documentation explaining how to add Amazon Sidewalk repository to nRF Connect SDK at http://..."

Reply:
added